pkg/testutil/testutil.go

```diff
@@ -37,6 +37,8 @@ import (
 	"gotest.tools/v3/icmd"
 )
 
+var FluentdImage = "fluent/fluentd:v1.14-1"
```

Comment:
This should be in `testutil_linux.go`

Reply:
ping @ktock for mirroring

Reply:
Mirrored: [`ghcr.io/stargz-containers/fluent/fluentd:v1.14-1-org`](https://github.com/orgs/stargz-containers/packages/container/package/fluent%2Ffluentd)
 cc @Zheaoli

Reply:
Got it